sdk/messaging/azeventhubs/README.md

```diff
@@ -98,36 +97,63 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
+* Frequently Asked Questions (FAQ) and Conceptual Topics in the detailed [Azure SDK for Go wiki][azure_sdk_for_go_wiki].
 
 <!-- ### Community-->
 ### Reporting security issues and security bugs
 
-Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue).
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter][security_techcenter].
 
 ### License
 
-Azure SDK for Go is licensed under the [MIT](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azeventhubs/LICENSE.txt) license.
+Azure SDK for Go is licensed under the [MIT][azeventhubs_mit_license] license.
 
 <!-- LINKS -->
+
+[azure_identity_pkg]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity
 [azure_sdk_for_go_contributing]: https://github.com/Azure/azure-sdk-for-go/blob/main/CONTRIBUTING.md
 [azure_sdk_for_go_contributing_developer_guide]: https://github.com/Azure/azure-sdk-for-go/blob/main/CONTRIBUTING.md#developer-guide
+[azure_sdk_for_go_contributing_first_issues]: https://github.com/azure/azure-sdk-for-go/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22
 [azure_sdk_for_go_contributing_pull_requests]: https://github.com/Azure/azure-sdk-for-go/blob/main/CONTRIBUTING.md#pull-requests
-
-[azure_identity_pkg]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity
+[azure_sdk_for_go_wiki]: https://github.com/azure/azure-sdk-for-go/wiki
+[azure_sub]: https://azure.microsoft.com/free/
+[azeventhubs_connectionstring_consumerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClientFromConnectionString
+[azeventhubs_connectionstring_producerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClientFromConnectionString
+[azeventhubs_consumerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#ConsumerClient
+[azeventhubs_mit_license]: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azeventhubs/LICENSE.txt
+[azeventhubs_repo]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/messaging/azeventhubs
+[azeventhubs_repo_main]: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azeventhubs
```

Comment:
Should these two be the same? They're in different parts of the document ... but shouldn't they be pointing to the same URL?  If not, please ignore. Thank you!